### PR TITLE
Add _pixel postfix to I<xx,yy,xy> and family

### LINKED ISSUE #397: Fix units for I[xx,yy,xy] and friends in Object Table
The second moments should be in arcseconds^2.  They're currently in pixels.

Multiple by pixel_scale^2 to correct.

### PR DESCRIPTION
**Pull request by yymao**
This PR add `_pixel` postfix to I<xx,yy,xy> and family, as discussed in #397. This PR fixes #397.

@wmwv do you have preference between `Ixx_pixel_r` and `Ixx_r_pixel`? Currently I chose the former.

### CODE REVIEW

File: GCRCatalogs/SCHEMA.md [hunk 1]
```diff
@@ -122,23 +122,21 @@ which is determined from the reference filter for each source (the filter that b
...
 `psFlux_<band>` | nJy | Point source model flux in `<band>.` |   | x |
 `psFluxErr_<band>` | nJy | Error value for `psFlux_<band>`. |   | x |
 `psFlux_flag_<band>` | - | Flag for issues with `psFlux_<band>`. |   | x |
-`Ixx_<band>` | asec2 | Adaptive second moment of the source intensity in `<band>`. |   | x |
-`Iyy_<band>` | asec2 | Adaptive second moment of the source intensity in `<band>`. |   | x |
-`Ixy_<band>` | asec2 | Adaptive second moment of the source intensity in `<band>`. |   | x |
-`IxxPSF_<band>` | asec2 | Adaptive second moment of the PSF  in `<band>`. |   | x |
-`IyyPSF_<band>` | asec2 | Adaptive second moment of the PSF  in `<band>`. |   | x |
-`IxyPSF_<band>` | asec2 | Adaptive second moment of the PSF  in `<band>`. |   | x |
+`Ixx_pixel_<band>` | pixel | Adaptive second moment of the source intensity in `<band>`. |   | xx |
```
Comment (wmwv) [inline GCRCatalogs/SCHEMA.md:134]: pixel^2

**Review by wmwv — APPROVED** [review]
Documentation and code changes look good to me.
I haven't run the code.

Only change suggested is that Ixx,yy,xy are in pixel^2.

**Comment by yymao** [comment]
Thanks @wmwv. I have tested the code. If interested, you can check out the resulting files on NERSC at

    /global/cscratch1/sd/yymao/desc/dc2_object_run2.2i_dr6_wfd_v1.new

**Comment by wmwv** [comment]
@yymao When I try to read in the full directory with Parquet (`pyarrow`), I get
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-21-53bee23e4dca> in <module>
      1 # Note that sampling_factor will not result in the same subsample for direct DPDD Parquet and the GCRCatalog subsample.
      2 if read_from_dpdd_parquet:
----> 3     df = pd.read_parquet(catalog_file, columns=columns, engine='pyarrow')  #, use_legacy_data=False)
      4     if sampling_factor > 1:
      5         df = df.sample(frac=1/sampling_factor)

/global/common/software/lsst/common/miniconda/py3.7-4.7.12.1-pyarrow/envs/desc/lib/python3.7/site-packages/pandas/io/parquet.py in read_parquet(path, engine, columns, **kwargs)
    310 
    311     impl = get_engine(engine)
--> 312     return impl.read(path, columns=columns, **kwargs)

/global/common/software/lsst/common/miniconda/py3.7-4.7.12.1-pyarrow/envs/desc/lib/python3.7/site-packages/pandas/io/parquet.py in read(self, path, columns, **kwargs)
    125         kwargs["use_pandas_metadata"] = True
    126         result = self.api.parquet.read_table(
--> 127             path, columns=columns, **kwargs
    128         ).to_pandas()
    129         if should_close:

~/local/lib/python3.7/site-packages/pyarrow/parquet.py in read_table(source, columns, use_threads, metadata, use_pandas_metadata, memory_map, read_dictionary, filesystem, filters, buffer_size, partitioning, use_legacy_dataset)
   1542                             buffer_size=buffer_size,
   1543                             filesystem=filesystem, filters=filters,
-> 1544                             partitioning=partitioning)
   1545     else:
   1546         pf = ParquetFile(source, metadata=metadata,

~/local/lib/python3.7/site-packages/pyarrow/parquet.py in __init__(self, path_or_paths, filesystem, schema, metadata, split_row_groups, validate_schema, filters, metadata_nthreads, read_dictionary, memory_map, buffer_size, partitioning, use_legacy_dataset)
   1201 
   1202         if validate_schema:
-> 1203             self.validate_schemas()
   1204 
   1205     def equals(self, other):

~/local/lib/python3.7/site-packages/pyarrow/parquet.py in validate_schemas(self)
   1250                                  '{!s}\n\nvs\n\n{!s}'
   1251                                  .format(piece, file_schema,
-> 1252                                          dataset_schema))
   1253 
   1254     def read(self, columns=None, use_threads=True, use_pandas_metadata=False):

ValueError: Schema in /global/cscratch1/sd/yymao/desc/dc2_object_run2.2i_dr6_wfd_v1.new/dc2_object_run2.2i_dr6_wfd_v1_tract2898.parquet was different. 
Iyy_pixel_r: double
  -- field metadata --
  PARQUET:field_id: '1'
magerr_g_cModel: double
  -- field metadata --
  PARQUET:field_id: '2'
magerr_y_cModel: double
  -- field metadata --
  PARQUET:field_id: '3'
x: double
  -- field metadata --
  PARQUET:field_id: '4'
psf_fwhm_z: double
  -- field metadata --
  PARQUET:field_id: '5'
cModelFluxErr_z: double
  -- field metadata --
  PARQUET:field_id: '6'
Ixx_pixel_u: double
  -- field metadata --
  PARQUET:field_id: '7'
Iyy_pixel_y: double
  -- field metadata --
  PARQUET:field_id: '8'
cModelFluxErr_u: double
  -- field metadata --
  PARQUET:field_id: '9'
psFluxErr_i: double
  -- field metadata --
  PARQUET:field_id: '10'
IyyPSF_pixel_u: double
  -- field metadata --
  PARQUET:field_id: '11'
I_flag_y: bool
  -- field metadata --
  PARQUET:field_id: '12'
Ixx_pixel_r: double
  -- field metadata --
  PARQUET:field_id: '13'
magerr_i: double
  -- field metadata --
  PARQUET:field_id: '14'
Ixy_pixel_g: double
  -- field metadata --
  PARQUET:field_id: '15'
I_flag_i: bool
  -- field metadata --
  PARQUET:field_id: '16'
IxyPSF_pixel_y: double
  -- field metadata --
  PARQUET:field_id: '17'
magerr_g: double
  -- field metadata --
  PARQUET:field_id: '18'
I_flag: bool
  -- field metadata --
  PARQUET:field_id: '19'
cModelFluxErr_g: double
  -- field metadata --
  PARQUET:field_id: '20'
magerr_i_cModel: double
  -- field metadata --
  PARQUET:field_id: '21'
snr_g_cModel: double
  -- field metadata --
  PARQUET:field_id: '22'
IyyPSF_pixel_g: double
  -- field metadata --
  PARQUET:field_id: '23'
I_flag_u: bool
  -- field metadata --
  PARQUET:field_id: '24'
cModelFlux_y: double
  -- field metadata --
  PARQUET:field_id: '25'
Iyy_pixel_u: double
  -- field metadata --
  PARQUET:field_id: '26'
mag_z: double
  -- field metadata --
  PARQUET:field_id: '27'
IxxPSF_pixel: double
  -- field metadata --
  PARQUET:field_id: '28'
cModelFlux_flag_y: bool
  -- field metadata --
  PARQUET:field_id: '29'
IxxPSF_pixel_i: double
  -- field metadata --
  PARQUET:field_id: '30'
I_flag_r: bool
  -- field metadata --
  PARQUET:field_id: '31'
Ixx_pixel_i: double
  -- field metadata --
  PARQUET:field_id: '32'
ra: double
  -- field metadata --
  PARQUET:field_id: '33'
IyyPSF_pixel: double
  -- field metadata --
  PARQUET:field_id: '34'
mag_i_cModel: double
  -- field metadata --
  PARQUET:field_id: '35'
mag_u_cModel: double
  -- field metadata --
  PARQUET:field_id: '36'
Iyy_pixel_g: double
  -- field metadata --
  PARQUET:field_id: '37'
clean: bool
  -- field metadata --
  PARQUET:field_id: '38'
IyyPSF_pixel_r: double
  -- field metadata --
  PARQUET:field_id: '39'
cModelFlux_u: double
  -- field metadata --
  PARQUET:field_id: '40'
y: double
  -- field metadata --
  PARQUET:field_id: '41'
magerr_y: double
  -- field metadata --
  PARQUET:field_id: '42'
mag_r: double
  -- field metadata --
  PARQUET:field_id: '43'
IxyPSF_pixel_g: double
  -- field metadata --
  PARQUET:field_id: '44'
Ixy_pixel_r: double
  -- field metadata --
  PARQUET:field_id: '45'
cModelFlux_flag_i: bool
  -- field metadata --
  PARQUET:field_id: '46'
cModelFlux_flag_z: bool
  -- field metadata --
  PARQUET:field_id: '47'
tract: int64
  -- field metadata --
  PARQUET:field_id: '48'
cModelFlux_flag_g: bool
  -- field metadata --
  PARQUET:field_id: '49'
Ixy_pixel_u: double
  -- field metadata --
  PARQUET:field_id: '50'
psFlux_g: double
  -- field metadata --
  PARQUET:field_id: '51'
psf_fwhm_g: double
  -- field metadata --
  PARQUET:field_id: '52'
mag_y_cModel: double
  -- field metadata --
  PARQUET:field_id: '53'
snr_y_cModel: double
  -- field metadata --
  PARQUET:field_id: '54'
parentObjectId: int64
  -- field metadata --
  PARQUET:field_id: '55'
psf_fwhm_y: double
  -- field metadata --
  PARQUET:field_id: '56'
psFluxErr_u: double
  -- field metadata --
  PARQUET:field_id: '57'
IxxPSF_pixel_u: double
  -- field metadata --
  PARQUET:field_id: '58'
psFlux_z: double
  -- field metadata --
  PARQUET:field_id: '59'
Iyy_pixel: double
  -- field metadata --
  PARQUET:field_id: '60'
psNdata: float
  -- field metadata --
  PARQUET:field_id: '61'
patch: string
  -- field metadata --
  PARQUET:field_id: '62'
IyyPSF_pixel_y: double
  -- field metadata --
  PARQUET:field_id: '63'
cModelFlux_z: double
  -- field metadata --
  PARQUET:field_id: '64'
IxyPSF_pixel: double
  -- field metadata --
  PARQUET:field_id: '65'
IxxPSF_pixel_y: double
  -- field metadata --
  PARQUET:field_id: '66'
IxyPSF_pixel_u: double
  -- field metadata --
  PARQUET:field_id: '67'
psFlux_i: double
  -- field metadata --
  PARQUET:field_id: '68'
cModelFluxErr_i: double
  -- field metadata --
  PARQUET:field_id: '69'
I_flag_g: bool
  -- field metadata --
  PARQUET:field_id: '70'
psFluxErr_z: double
  -- field metadata --
  PARQUET:field_id: '71'
psFluxErr_y: double
  -- field metadata --
  PARQUET:field_id: '72'
yErr: float
  -- field metadata --
  PARQUET:field_id: '73'
IxyPSF_pixel_r: double
  -- field metadata --
  PARQUET:field_id: '74'
cModelFlux_r: double
  -- field metadata --
  PARQUET:field_id: '75'
cModelFlux_g: double
  -- field metadata --
  PARQUET:field_id: '76'
Ixx_pixel_z: double
  -- field metadata --
  PARQUET:field_id: '77'
psFluxErr_g: double
  -- field metadata --
  PARQUET:field_id: '78'
cModelFluxErr_r: double
  -- field metadata --
  PARQUET:field_id: '79'
psFlux_y: double
  -- field metadata --
  PARQUET:field_id: '80'
psFlux_flag_y: bool
  -- field metadata --
  PARQUET:field_id: '81'
xy_flag: bool
  -- field metadata --
  PARQUET:field_id: '82'
cModelFluxErr_y: double
  -- field metadata --
  PARQUET:field_id: '83'
psFlux_flag_g: bool
  -- field metadata --
  PARQUET:field_id: '84'
psFlux_flag_r: bool
  -- field metadata --
  PARQUET:field_id: '85'
Ixx_pixel: double
  -- field metadata --
  PARQUET:field_id: '86'
extendedness: double
  -- field metadata --
  PARQUET:field_id: '87'
IxxPSF_pixel_g: double
  -- field metadata --
  PARQUET:field_id: '88'
mag_u: double
  -- field metadata --
  PARQUET:field_id: '89'
magerr_u: double
  -- field metadata --
  PARQUET:field_id: '90'
IxxPSF_pixel_r: double
  -- field metadata --
  PARQUET:field_id: '91'
mag_i: double
  -- field metadata --
  PARQUET:field_id: '92'
good: bool
  -- field metadata --
  PARQUET:field_id: '93'
psFlux_u: double
  -- field metadata --
  PARQUET:field_id: '94'
magerr_r: double
  -- field metadata --
  PARQUET:field_id: '95'
dec: double
  -- field metadata --
  PARQUET:field_id: '96'
psFlux_flag_u: bool
  -- field metadata --
  PARQUET:field_id: '97'
IyyPSF_pixel_i: double
  -- field metadata --
  PARQUET:field_id: '98'
mag_y: double
  -- field metadata --
  PARQUET:field_id: '99'
cModelFlux_flag_r: bool
  -- field metadata --
  PARQUET:field_id: '100'
Iyy_pixel_z: double
  -- field metadata --
  PARQUET:field_id: '101'
magerr_z_cModel: double
  -- field metadata --
  PARQUET:field_id: '102'
cModelFlux_flag_u: bool
  -- field metadata --
  PARQUET:field_id: '103'
psFlux_flag_i: bool
  -- field metadata --
  PARQUET:field_id: '104'
xErr: float
  -- field metadata --
  PARQUET:field_id: '105'
IxxPSF_pixel_z: double
  -- field metadata --
  PARQUET:field_id: '106'
cModelFlux_i: double
  -- field metadata --
  PARQUET:field_id: '107'
I_flag_z: bool
  -- field metadata --
  PARQUET:field_id: '108'
psFluxErr_r: double
  -- field metadata --
  PARQUET:field_id: '109'
psf_fwhm_i: double
  -- field metadata --
  PARQUET:field_id: '110'
blendedness: double
  -- field metadata --
  PARQUET:field_id: '111'
mag_r_cModel: double
  -- field metadata --
  PARQUET:field_id: '112'
Ixy_pixel: double
  -- field metadata --
  PARQUET:field_id: '113'
snr_z_cModel: double
  -- field metadata --
  PARQUET:field_id: '114'
psFlux_r: double
  -- field metadata --
  PARQUET:field_id: '115'
IxyPSF_pixel_z: double
  -- field metadata --
  PARQUET:field_id: '116'
snr_r_cModel: double
  -- field metadata --
  PARQUET:field_id: '117'
Ixx_pixel_g: double
  -- field metadata --
  PARQUET:field_id: '118'
magerr_r_cModel: double
  -- field metadata --
  PARQUET:field_id: '119'
Iyy_pixel_i: double
  -- field metadata --
  PARQUET:field_id: '120'
Ixy_pixel_y: double
  -- field metadata --
  PARQUET:field_id: '121'
mag_g: double
  -- field metadata --
  PARQUET:field_id: '122'
psFlux_flag_z: bool
  -- field metadata --
  PARQUET:field_id: '123'
snr_u_cModel: double
  -- field metadata --
  PARQUET:field_id: '124'
Ixy_pixel_i: double
  -- field metadata --
  PARQUET:field_id: '125'
IyyPSF_pixel_z: double
  -- field metadata --
  PARQUET:field_id: '126'
magerr_u_cModel: double
  -- field metadata --
  PARQUET:field_id: '127'
mag_z_cModel: double
  -- field metadata --
  PARQUET:field_id: '128'
IxyPSF_pixel_i: double
  -- field metadata --
  PARQUET:field_id: '129'
mag_g_cModel: double
  -- field metadata --
  PARQUET:field_id: '130'
psf_fwhm_u: double
  -- field metadata --
  PARQUET:field_id: '131'
Ixx_pixel_y: double
  -- field metadata --
  PARQUET:field_id: '132'
Ixy_pixel_z: double
  -- field metadata --
  PARQUET:field_id: '133'
objectId: int64
  -- field metadata --
  PARQUET:field_id: '134'
psf_fwhm_r: double
  -- field metadata --
  PARQUET:field_id: '135'
magerr_z: double
  -- field metadata --
  PARQUET:field_id: '136'
snr_i_cModel: double
  -- field metadata --
  PARQUET:field_id: '137'
-- schema metadata --
ARROW:schema: '/////5gcAAAQAAAAAAAKAAwABgAFAAgACgAAAAABAwAMAAAACAAIAAAABA' + 9714

vs

snr_y_cModel: double
  -- field metadata --
  PARQUET:field_id: '1'
I_flag_y: bool
  -- field metadata --
  PARQUET:field_id: '2'
clean: bool
  -- field metadata --
  PARQUET:field_id: '3'
mag_y_cModel: double
  -- field metadata --
  PARQUET:field_id: '4'
Ixx_pixel_z: double
  -- field metadata --
  PARQUET:field_id: '5'
Iyy_pixel_z: double
  -- field metadata --
  PARQUET:field_id: '6'
psFlux_u: double
  -- field metadata --
  PARQUET:field_id: '7'
magerr_g: double
  -- field metadata --
  PARQUET:field_id: '8'
mag_g: double
  -- field metadata --
  PARQUET:field_id: '9'
cModelFluxErr_u: double
  -- field metadata --
  PARQUET:field_id: '10'
IxxPSF_pixel_y: double
  -- field metadata --
  PARQUET:field_id: '11'
psFlux_flag_i: bool
  -- field metadata --
  PARQUET:field_id: '12'
psFlux_flag_g: bool
  -- field metadata --
  PARQUET:field_id: '13'
magerr_u: double
  -- field metadata --
  PARQUET:field_id: '14'
patch: string
  -- field metadata --
  PARQUET:field_id: '15'
I_flag: bool
  -- field metadata --
  PARQUET:field_id: '16'
xy_flag: bool
  -- field metadata --
  PARQUET:field_id: '17'
mag_i_cModel: double
  -- field metadata --
  PARQUET:field_id: '18'
psNdata: float
  -- field metadata --
  PARQUET:field_id: '19'
Ixy_pixel_u: double
  -- field metadata --
  PARQUET:field_id: '20'
psFlux_z: double
  -- field metadata --
  PARQUET:field_id: '21'
IyyPSF_pixel_z: double
  -- field metadata --
  PARQUET:field_id: '22'
cModelFlux_flag_i: bool
  -- field metadata --
  PARQUET:field_id: '23'
Ixx_pixel_r: double
  -- field metadata --
  PARQUET:field_id: '24'
IxyPSF_pixel: double
  -- field metadata --
  PARQUET:field_id: '25'
y: double
  -- field metadata --
  PARQUET:field_id: '26'
xErr: float
  -- field metadata --
  PARQUET:field_id: '27'
Ixx_pixel_u: double
  -- field metadata --
  PARQUET:field_id: '28'
mag_z_cModel: double
  -- field metadata --
  PARQUET:field_id: '29'
cModelFlux_u: double
  -- field metadata --
  PARQUET:field_id: '30'
Iyy_pixel_i: double
  -- field metadata --
  PARQUET:field_id: '31'
psFluxErr_r: double
  -- field metadata --
  PARQUET:field_id: '32'
cModelFlux_g: double
  -- field metadata --
  PARQUET:field_id: '33'
Iyy_pixel_y: double
  -- field metadata --
  PARQUET:field_id: '34'
Iyy_pixel_g: double
  -- field metadata --
  PARQUET:field_id: '35'
IxyPSF_pixel_r: double
  -- field metadata --
  PARQUET:field_id: '36'
cModelFluxErr_z: double
  -- field metadata --
  PARQUET:field_id: '37'
psFlux_flag_y: bool
  -- field metadata --
  PARQUET:field_id: '38'
cModelFlux_r: double
  -- field metadata --
  PARQUET:field_id: '39'
Ixy_pixel_g: double
  -- field metadata --
  PARQUET:field_id: '40'
cModelFlux_flag_u: bool
  -- field metadata --
  PARQUET:field_id: '41'
snr_u_cModel: double
  -- field metadata --
  PARQUET:field_id: '42'
tract: int64
  -- field metadata --
  PARQUET:field_id: '43'
mag_u_cModel: double
  -- field metadata --
  PARQUET:field_id: '44'
IxxPSF_pixel_u: double
  -- field metadata --
  PARQUET:field_id: '45'
x: double
  -- field metadata --
  PARQUET:field_id: '46'
magerr_y_cModel: double
  -- field metadata --
  PARQUET:field_id: '47'
IxxPSF_pixel: double
  -- field metadata --
  PARQUET:field_id: '48'
IxyPSF_pixel_i: double
  -- field metadata --
  PARQUET:field_id: '49'
Iyy_pixel_u: double
  -- field metadata --
  PARQUET:field_id: '50'
psFluxErr_i: double
  -- field metadata --
  PARQUET:field_id: '51'
cModelFluxErr_r: double
  -- field metadata --
  PARQUET:field_id: '52'
mag_y: double
  -- field metadata --
  PARQUET:field_id: '53'
psFlux_flag_u: bool
  -- field metadata --
  PARQUET:field_id: '54'
psFluxErr_u: double
  -- field metadata --
  PARQUET:field_id: '55'
extendedness: double
  -- field metadata --
  PARQUET:field_id: '56'
psf_fwhm_g: double
  -- field metadata --
  PARQUET:field_id: '57'
Ixy_pixel_y: double
  -- field metadata --
  PARQUET:field_id: '58'
mag_r: double
  -- field metadata --
  PARQUET:field_id: '59'
magerr_z: double
  -- field metadata --
  PARQUET:field_id: '60'
magerr_i: double
  -- field metadata --
  PARQUET:field_id: '61'
yErr: float
  -- field metadata --
  PARQUET:field_id: '62'
IyyPSF_pixel_i: double
  -- field metadata --
  PARQUET:field_id: '63'
Ixx_pixel: double
  -- field metadata --
  PARQUET:field_id: '64'
dec: double
  -- field metadata --
  PARQUET:field_id: '65'
mag_z: double
  -- field metadata --
  PARQUET:field_id: '66'
cModelFluxErr_g: double
  -- field metadata --
  PARQUET:field_id: '67'
magerr_z_cModel: double
  -- field metadata --
  PARQUET:field_id: '68'
psf_fwhm_r: double
  -- field metadata --
  PARQUET:field_id: '69'
ra: double
  -- field metadata --
  PARQUET:field_id: '70'
IxxPSF_pixel_r: double
  -- field metadata --
  PARQUET:field_id: '71'
IxxPSF_pixel_i: double
  -- field metadata --
  PARQUET:field_id: '72'
magerr_i_cModel: double
  -- field metadata --
  PARQUET:field_id: '73'
Iyy_pixel_r: double
  -- field metadata --
  PARQUET:field_id: '74'
I_flag_g: bool
  -- field metadata --
  PARQUET:field_id: '75'
psf_fwhm_y: double
  -- field metadata --
  PARQUET:field_id: '76'
I_flag_z: bool
  -- field metadata --
  PARQUET:field_id: '77'
blendedness: double
  -- field metadata --
  PARQUET:field_id: '78'
snr_g_cModel: double
  -- field metadata --
  PARQUET:field_id: '79'
Ixy_pixel_i: double
  -- field metadata --
  PARQUET:field_id: '80'
psFlux_flag_z: bool
  -- field metadata --
  PARQUET:field_id: '81'
Ixy_pixel_z: double
  -- field metadata --
  PARQUET:field_id: '82'
psFlux_r: double
  -- field metadata --
  PARQUET:field_id: '83'
Ixx_pixel_y: double
  -- field metadata --
  PARQUET:field_id: '84'
cModelFlux_i: double
  -- field metadata --
  PARQUET:field_id: '85'
psFluxErr_z: double
  -- field metadata --
  PARQUET:field_id: '86'
Ixy_pixel_r: double
  -- field metadata --
  PARQUET:field_id: '87'
IyyPSF_pixel_y: double
  -- field metadata --
  PARQUET:field_id: '88'
parentObjectId: int64
  -- field metadata --
  PARQUET:field_id: '89'
cModelFluxErr_i: double
  -- field metadata --
  PARQUET:field_id: '90'
psFlux_flag_r: bool
  -- field metadata --
  PARQUET:field_id: '91'
mag_r_cModel: double
  -- field metadata --
  PARQUET:field_id: '92'
magerr_y: double
  -- field metadata --
  PARQUET:field_id: '93'
snr_z_cModel: double
  -- field metadata --
  PARQUET:field_id: '94'
cModelFlux_z: double
  -- field metadata --
  PARQUET:field_id: '95'
psf_fwhm_i: double
  -- field metadata --
  PARQUET:field_id: '96'
psFluxErr_y: double
  -- field metadata --
  PARQUET:field_id: '97'
magerr_r_cModel: double
  -- field metadata --
  PARQUET:field_id: '98'
cModelFlux_y: double
  -- field metadata --
  PARQUET:field_id: '99'
cModelFlux_flag_r: bool
  -- field metadata --
  PARQUET:field_id: '100'
snr_i_cModel: double
  -- field metadata --
  PARQUET:field_id: '101'
IxxPSF_pixel_g: double
  -- field metadata --
  PARQUET:field_id: '102'
magerr_g_cModel: double
  -- field metadata --
  PARQUET:field_id: '103'
mag_g_cModel: double
  -- field metadata --
  PARQUET:field_id: '104'
snr_r_cModel: double
  -- field metadata --
  PARQUET:field_id: '105'
I_flag_u: bool
  -- field metadata --
  PARQUET:field_id: '106'
cModelFlux_flag_g: bool
  -- field metadata --
  PARQUET:field_id: '107'
Ixy_pixel: double
  -- field metadata --
  PARQUET:field_id: '108'
IxyPSF_pixel_z: double
  -- field metadata --
  PARQUET:field_id: '109'
psFlux_i: double
  -- field metadata --
  PARQUET:field_id: '110'
psFluxErr_g: double
  -- field metadata --
  PARQUET:field_id: '111'
psFlux_g: double
  -- field metadata --
  PARQUET:field_id: '112'
cModelFlux_flag_z: bool
  -- field metadata --
  PARQUET:field_id: '113'
I_flag_r: bool
  -- field metadata --
  PARQUET:field_id: '114'
psf_fwhm_z: double
  -- field metadata --
  PARQUET:field_id: '115'
psFlux_y: double
  -- field metadata --
  PARQUET:field_id: '116'
I_flag_i: bool
  -- field metadata --
  PARQUET:field_id: '117'
Iyy_pixel: double
  -- field metadata --
  PARQUET:field_id: '118'
cModelFluxErr_y: double
  -- field metadata --
  PARQUET:field_id: '119'
IxyPSF_pixel_u: double
  -- field metadata --
  PARQUET:field_id: '120'
cModelFlux_flag_y: bool
  -- field metadata --
  PARQUET:field_id: '121'
IyyPSF_pixel: double
  -- field metadata --
  PARQUET:field_id: '122'
Ixx_pixel_i: double
  -- field metadata --
  PARQUET:field_id: '123'
IyyPSF_pixel_r: double
  -- field metadata --
  PARQUET:field_id: '124'
IxxPSF_pixel_z: double
  -- field metadata --
  PARQUET:field_id: '125'
mag_i: double
  -- field metadata --
  PARQUET:field_id: '126'
mag_u: double
  -- field metadata --
  PARQUET:field_id: '127'
IyyPSF_pixel_g: double
  -- field metadata --
  PARQUET:field_id: '128'
magerr_r: double
  -- field metadata --
  PARQUET:field_id: '129'
IxyPSF_pixel_g: double
  -- field metadata --
  PARQUET:field_id: '130'
IxyPSF_pixel_y: double
  -- field metadata --
  PARQUET:field_id: '131'
objectId: int64
  -- field metadata --
  PARQUET:field_id: '132'
Ixx_pixel_g: double
  -- field metadata --
  PARQUET:field_id: '133'
IyyPSF_pixel_u: double
  -- field metadata --
  PARQUET:field_id: '134'
psf_fwhm_u: double
  -- field metadata --
  PARQUET:field_id: '135'
magerr_u_cModel: double
  -- field metadata --
  PARQUET:field_id: '136'
good: bool
  -- field metadata --
  PARQUET:field_id: '137'
-- schema metadata --
ARROW:schema: '/////5gcAAAQAAAAAAAKAAwABgAFAAgACgAAAAABAwAMAAAACAAIAAAABA' + 9714
```

**Comment by wmwv** [comment]
Is this a problem in `pyarrow` or a true inconsistency in the schema?

**Comment by yymao** [comment]
Hmm @wmwv -- I tried checking the schema across files using dask and got consistent results. I'll try with pyarrow now.

```python
import dask.dataframe
dir_path = "/global/cscratch1/sd/yymao/desc/dc2_object_run2.2i_dr6_wfd_v1.new"
ddf = dask.dataframe.read_parquet(dir_path + "/*.parquet")
schema = dict(ddf.get_partition(0).dtypes)
for n in range(ddf.npartitions):
    assert schema == dict(ddf.get_partition(n).dtypes), "Schema differ for {}".format(n)
```

**Comment by yymao** [comment]
Oh I think I know what's wrong, the orders of columns are different, but as a dictionary they are evaluated as the same.

**Comment by yymao** [comment]
Fixing this now in DC2-production

**Comment by yymao** [comment]
Fix proposed in https://github.com/LSSTDESC/DC2-production/pull/411

**Comment by yymao** [comment]
Newly generated files are in `/global/cfs/cdirs/lsst/shared/DC2-prod/Run2.2i/dpdd/Run2.2i-dr6-wfd-v1/object_dpdd_only`

**Review by JoanneBogart — APPROVED** [review]
LGTM.  I confirm that column names for Ixx,... are formed as advertised.

**Pull request merged**